assets/css/less/wordpress-default-themes.less

```diff
@@ -106,29 +107,18 @@
 		input[type="submit"],
 		select,
 		textarea {
-			padding: 8px;
+			padding: @padding-input-wp-current-themes;
 			max-width: 100%;
 			font-size: inherit;
 			font-family: inherit;
 		}
 
-		.wpbdp-button,
-		input[type="submit"] {
-			color: var(--wp--preset--color--background);
```

Comment:
Was there a reason to remove this?

Reply:
We need a white color for buttons, similar to the following image:
<img width="869" alt="image" src="https://user-images.githubusercontent.com/69119241/230505841-3c8136bb-d8b1-4add-920b-16a7147e9cc6.png">

Reply:
If the setting to override the buttons is off, this css should be used. Can you please attach a screenshot of these buttons in the current version and in this version with the override button setting off?

Reply:
<img width="813" alt="image" src="https://user-images.githubusercontent.com/69119241/230508177-64d1a8d8-7eaf-481c-a331-55269bb9ee04.png">

Reply:
![image](https://user-images.githubusercontent.com/69119241/230508224-cd4ecc9e-633a-4aa1-a96b-c89e0358a809.png)

Reply:
Thanks! When that setting is off, the button should have the same colors as before.

Reply:
<img width="319" alt="image" src="https://user-images.githubusercontent.com/69119241/230511826-db877d68-0aa1-4b0c-b286-9370b7aa1c43.png">

<img width="820" alt="image" src="https://user-images.githubusercontent.com/69119241/230511803-75ffb709-767b-42e7-accd-1ad60b0d840d.png">

---

<img width="340" alt="image" src="https://user-images.githubusercontent.com/69119241/230511893-cecaf015-2ab5-4b73-ba4d-62acc3eb75c0.png">

<img width="710" alt="image" src="https://user-images.githubusercontent.com/69119241/230511880-94eaa3eb-5a0e-4b93-99a3-51b49ef52d05.png">

Reply:
Hmm... It looks like the "Add listing" and "Directory" buttons were shown as links before. Can we fix that too? I don't think it's an improvement to change all of those to buttons.

Reply:
@stephywells, we have already styled the WP 2022 theme as shown below. Are you suggesting that we only remove button styles for Search Box Links in the WP 2023 theme?

### WP 2022 theme - BD Core 6.3.3 - Disable Button Style:
<img width="775" alt="image" src="https://user-images.githubusercontent.com/69119241/230516929-f7e854b3-8664-49e3-b7b4-14bb61af52f5.png">

### WP 2022 theme - BD Core 6.3.3 -  Enable Button Style:
<img width="729" alt="image" src="https://user-images.githubusercontent.com/69119241/230517025-21cb366b-b72b-4134-9c13-d2cfa56c7813.png">

Reply:
@shervElmi I'm referring to the before screenshot here:
<img width="355" alt="image" src="https://user-images.githubusercontent.com/69119241/230508362-e88b98b9-7531-4880-ab17-9a64f0bca62e.png">

"Directory" and "Add listing" are links there instead of buttons. It feels weird to have all three of those as primary buttons, especially since they are taller than the "find listings" button. In this demo, the buttons are smaller:
https://demo.businessdirectoryplugin.com/

So how about this?
- Remove the "button wpbdp-button" class on all those links except "Add Listing"
- Use a secondary button style for "add listing" (the button color is used for the border and text color with a transparent background. On hover, reverse the colors.)
- Vertically center them using flex and flex gap:
```
.wpbdp-main-links {
display: flex;
justify-content: flex-end;
gap: 12px; (I'm not sure what's the standard gap size)
align-items: center;
}
```